gravitee-apim-portal-webui-next/src/components/api-picture/api-picture.component.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input } from '@angular/core';
+import { toSvg } from 'jdenticon';
+
+@Component({
+  selector: 'app-api-picture',
+  standalone: true,
+  imports: [],
+  template: `
+    <img
+      i18n-alt="@@apiPicture"
+      alt="API Picture"
+      class="round"
+      [src]="picture"
+      [height]="size"
+      [width]="size"
+      (error)="useGeneratedPicture()"
+      placeholder="assets/images/logo.png" />
+  `,
+  styleUrl: './api-picture.component.scss',
+})
+export class ApiPictureComponent {
+  @Input()
+  name!: string;
+  @Input()
+  version!: string;
```

Comment:
Do we need `!` since we already have in tsconfig `strictPropertyInitialization`

Reply:
Just found out that we don't have `strictPropertyInitialization` anymore 😅 all good then

Reply:
I removed the tsconfig setting `strictPropertyInitialization` to make it clearer in our code when a variable is required or not. I will add though a `{required: true}` to the Input